scripts/run.sh

```
@@ -72,7 +72,7 @@ if [ -z "$SLURM_ARRAY_TASK_ID" ]; then
     fi
 fi
 
-source env/bin/activate
+source /w/20252/khashazad/env/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /w/20252/khashazad/env/bin/activate was not specified as input (see shellcheck -x).

(SC1091)